src/devices/Mcp960X/README.md

```diff
@@ -0,0 +1,13 @@
+﻿# Mcp960X - cold-junction compensated thermocouple to digital converter
+
+## Summary
```

Comment:
please add summary directly under the title, then links to the documentation then add tiny usage sample. See i.e. https://github.com/dotnet/iot/tree/main/src/devices/Apa102 for reference

We're currently in the process of unifying the docs format

Reply:
README.md changed accroding to review commet and Apa102 example